libs/media-player/src/lib/MediaPlayerButton.tsx

```diff
@@ -0,0 +1,7 @@
+import { Box } from '@mui/material';
```

Comment:
This reminds me that I still need to make sure we inject the MediaPlayerButton and export a default MediaPlayerButton that is themeable.